src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__container {
```

Comment:
According to the checklist, each block should have its own CSS file. Here, all the styles are placed in the single file.

Reply:
Reworked each block in its own CSS file.